site/src/components/Resources/AgentStatus.tsx

```diff
@@ -58,15 +58,15 @@ const ConnectingStatus: React.FC = () => {
 
 const TimeoutStatus: React.FC<{
   agent: WorkspaceAgent
-  workspace: Workspace
-}> = ({ agent, workspace }) => {
+}> = ({ agent }) => {
   const { t } = useTranslation("agent")
   const styles = useStyles()
   const anchorRef = useRef<SVGSVGElement>(null)
   const [isOpen, setIsOpen] = useState(false)
   const id = isOpen ? "timeout-popover" : undefined
   const troubleshootLink =
-    agent.troubleshooting_url ?? `/templates/${workspace.template_name}#readme`
+    agent.troubleshooting_url ??
+    "https://coder.com/docs/coder-oss/latest/templates#troubleshooting-templates"
```

Comment:
How about we change this in the API so that it will always return a (this) troubleshooting URL if none is configured? We could even expose a CLI option to allow admins to manually define the (fallback) troubleshooting URL.

Reply:
Ooh, I'm in favor of doing both. Perhaps we make the CLI option hidden since it's a quite "niche" feature.

Reply:
Great, I'll update #5005 to make it so 👍🏻

Reply:
@bpmct, I had to implement some of your changes here in my #5005 PR so that tests/lints pass.

Reply:
I'm a bit confused. Should we close this in favor of #5005 which will cover both the CLI and dashboard?

Reply:
I didn't do the docs or i18n changes, so those are totally valid, but I had to implement the changes in `AgentStatus.tsx` in my PR as well.